### DISCUSSION TITLE
How can I remove a blob from dashboard?

### DISCUSSION THREAD

**Discussion by thjiang** [vercel/community#4997] [Help] [answered]
### Summary

I need to delete a store, but the console does not let me delete it. The prompt is as follows:

<img width="768" alt="image" src="https://github.com/vercel/community/assets/18193558/d2605cee-143f-408c-a107-de5905ac04e5">

<img width="951" alt="image" src="https://github.com/vercel/community/assets/18193558/80cc3c8e-f403-4d4a-b24e-bcff2822fabd">

### Steps to Reproduce

https://vercel.com/dashboard/xxxx/stores
setting
Delete Store

**Comment by swarnava** [comment]
Hey,

You can write a code using Vercel SDK to iterate through every blob and execute a delete command to remove blobs first before deleting the blob store. 

Thanks!

**Reply by thjiang** [reply]
Thanks for your help.

When I try to list blobs by the Vercel SDK I got an error:  
Vercel Blob: This store has been suspended

Maybe because I have reached the 100M outgoing bandwidth usage limits?

Is there any other way I can delete this blob?

**Reply by davincho** [reply]
I have the same issue, seems like the limitation also applies to requests coming through the dashboard 🤔

**Reply by danielo515** [reply]
I am in the same situation. I am not interested in the blob anymore, and I want to delete it, but I can't because quotas

**Comment by thjiang** [comment]
A month has passed, and this blob is still in a locked state. Is there any way to solve it?

**Comment by amyegan** [comment, marked as answer]
Hi everyone! We made a change that will let you view the list and delete blobs from the dashboard even after limits are exceeded. Please give it a try and let me know if it works for you 🙏

**Reply by thjiang** [reply]
Thanks for your help, I can delete it from the dashboard now.

<img width="1003" alt="zm20240428-1328-2" src="https://github.com/vercel/community/assets/18193558/d08b663a-7178-4973-a50b-bbead74fd386">
